shell/platform/windows/angle_surface_manager.cc

```diff
@@ -288,4 +317,24 @@ EGLBoolean AngleSurfaceManager::SwapBuffers() {
   return (eglSwapBuffers(egl_display_, render_surface_));
 }
 
+bool AngleSurfaceManager::DirectCompositionSupported() {
+#ifdef WINUWP
+  return true;
```

Comment:
DComp isn't supported for UWP

Reply:
This PR should not affect UWP in any way, the behavior for UWP should remain completely identical. I will test it though.

As for the method, I think maybe `DirectCompositionSupported` is not the bast name for it. The semantic pretty much matches `IsNewerThanWindows7`.  So maybe `IsCompositionSupported` would be better?

Reply:
Confirmed that UWP is unaffected and understand the logic.  CompositionSupported would be better, yes.  You should add some comments explaining what's going on and why we need the surface create / destroy when composition not supported.